android/src/main/java/com/rudderstack/react/android/RNRudderSdkModule.java

```diff
@@ -71,11 +77,14 @@ public void setup(ReadableMap options, ReadableMap rudderOptionsMap, Promise pro
             }
 
             if (options.hasKey("trackAppLifecycleEvents")) {
+                trackLifeCycleEvents = options.getBoolean("trackAppLifecycleEvents");
                 configBuilder.withTrackLifecycleEvents(options.getBoolean("trackAppLifecycleEvents"));
             }
             if (options.hasKey("recordScreenViews")) {
-                configBuilder.withRecordScreenViews(options.getBoolean("recordScreenViews"));
+                recordScreenViews = options.getBoolean("recordScreenViews");
             }
+            configBuilder.withRecordScreenViews(false);
```

Comment:
@desusai7 why `false` is hardcoded here?

Reply:
We had hardcoded the value as false here since we are tracking the automatic screen recording in the RNLifeCycleEventListener.java of RN Wrapper and we don't want the native Android SDK to track the screen views again.

Will add a comment as well there to make it more clear

Reply:
Got it. Thanks.

Reply:
Added comment